Automattic/wp-calypso
# Settings/Framework: Remove HelloVote setting and notice

Removes the HelloVote setting introduced in #8436 and notice introduced in #8423 (plus refinements in a few follow-up PRs).

### Screenshot: Setting

![image](https://cloud.githubusercontent.com/assets/227022/19062466/63dd2bea-89c8-11e6-9cbe-3ef566efe97c.png)

### Screenshot: Notice

![image](https://cloud.githubusercontent.com/assets/227022/19125497/467af2a6-8b05-11e6-8198-014e5685aa3a.png)

### To test

Visit `/settings/general/$site` for a non-Jetpack site.  Verify that the setting disappears and the other settings still work.

Visit almost any site-specific page for a non-Jetpack site (editor, posts listing for example).  Verify that the HelloVote notice is not active and the main layout is not broken.  See #8423 for more detailed testing instructions for this feature.

### Note

There's still one instance of HelloVote-related code: the [`helloVoteDismissed` user preference](https://github.com/Automattic/wp-calypso/blob/f6127fb/client/state/preferences/schema.js#L18) in the user preferences schema.  I left this here to avoid unnecessarily invalidating saved state.
